Fix(eos_designs)!: Remove eBGP LAN outbound route-map for WAN (#4107)
aristanetworks/avd: ansible_collections/arista/avd/docs/release-notes/4.x.x.md

- Documentation for AVD version `4.x.x` [available here](https://www.avd.sh/en/stable/)

<!-- Release notes generated using configuration in .github/release.yml at devel -->
## Release 4.9.0

### Breaking or behavioral changes in eos_designs

Breaking changes may require modifications to the inventory or playbook.

#### Removed the BGP outbound route-map for eBGP LAN on WAN routers

The eBGP LAN feature for WAN routers introduced in AVD 4.7.0 is not working for connected and static redistributed routes inside a VRF (outside of the VRF default) as the RM-BGP-UNDERLAY-PEERS-OUT outbound route-map would not match these routes.
Similarly the routes coming from an eBGP peering in any VRF (including default) would be blocked out by the route-map.

> For all these reasons, the route-map is being removed.

Reviewer: ```suggestion
For these reasons, the route-map was removed as a bug fix.
```

> To keep the broken route-map, structured_configuration can be used.

Reviewer: ```suggestion
To keep the broken route-map, `structured_config` can be used.
```
